x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/text_expansion_callout.tsx

```diff
@@ -182,6 +185,94 @@ export const ModelDeploymentInProgress = ({
 export const ModelDeployed = ({
```

Comment:
question: this file is now 350 lines long and contains five different components. Have you considered splitting it up?

Reply:
Here's the ticket: https://github.com/elastic/enterprise-search-team/issues/4440